website_sale_add_to_cart_popup/controllers/product_configurator.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2021 Coop IT Easy SCRLfs <https://coopiteasy.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.http import request
+
+from odoo.addons.sale_product_configurator.controllers.main import (
+    ProductConfiguratorController,
+)
+
+
+class ProductConfiguratorController(ProductConfiguratorController):
+    def _show_optional_products(
+        self, product_id, variant_values, pricelist, handle_stock, **kw
+    ):
+        """
+        Overwrite default behaviour to always show a content for the
+        add to cart modal.
+
+        Copied verbatim from sale_product_configurator, and slightly adjusted.
+        """
+        product = request.env["product.product"].browse(int(product_id))
+        combination = request.env["product.template.attribute.value"].browse(
+            variant_values
+        )
+        # Commented out from upstream. Preserved for posterity's sake.
+        #
+        # has_optional_products = product.optional_product_ids.filtered(
+        #     lambda p: p._is_add_to_cart_possible(combination)
+        # )
+        #
+        # if not has_optional_products:
+        #     return False
+
+        add_qty = int(kw.get("add_qty", 1))
+
+        no_variant_attribute_values = combination.filtered(
+            lambda product_template_attribute_value: product_template_attribute_value.attribute_id.create_variant  # noqa: B950
```

Comment:
why the super long variable name? i like explicit, but here i think it hinders the readability.

Reply:
Suggestion, using `product_tmpl_att_value` instead of `product_template_attribute_value`.

Reply:
Done :+1: